lib/MetaCPAN/Web/Controller/Sysadmin/Datacenters.pm

```diff
@@ -0,0 +1,29 @@
+package MetaCPAN::Web::Controller::Sysadmin::Datacenters;
```

Comment:
Could we s/Sysadmin::Datacenters/SysAdmin::DataCenters/?  I know it's picky, but if we do this later a lot of people with checkouts on OS X are going to be irritated by the change in case. ;)